static/js/muting.js

```diff
@@ -51,6 +52,50 @@ exports.set_muted_topics = function (tuples) {
     });
 };
 
+exports.mute_notification = (function () {
```

Comment:
I still think this should live in muting_ui.js.  I might not have explained my earlier suggestion clearly.  Can you try to catch me on Zulip?